test/format/verify.c

```diff
@@ -191,7 +234,25 @@ table_verify_mirror(WT_CONNECTION *conn, TABLE *base, TABLE *table, const char *
       table->id, checkpoint == NULL ? "" : checkpoint, checkpoint == NULL ? "" : " checkpoint ");
     trace_msg(session, "%s: start", buf);
 
-    for (failures = 0, rows = 1; rows <= TV(RUNS_ROWS); ++rows) {
+    /*
+     * By default compare the entire range of keys, however if thread info is provided the start/end
+     * key ranges can be used instead. These ranges follow the same rules as truncate; If the
+     * provided value is zero treat that as the start/end of the table.
+     */
+    range_begin = 1;
+    range_end = TV(RUNS_ROWS);
+    if (tinfo != NULL) {
+        if (tinfo->keyno != 0) {
+            range_begin = tinfo->keyno;
+            testutil_check(position_cursor_before(base, base_cursor, range_begin));
+            testutil_check(position_cursor_before(table, table_cursor, range_begin));
+        }
+
+        if (tinfo->last != 0)
+            range_end = tinfo->last;
```

Comment:
>I wonder if this tinfo->keyno or tinfo->last ever gets reset after the operation thread finishes their operation?

I worked through the references to these fields and the only time we set them is inside [this `if (op == TRUNCATE) {`](https://github.com/wiredtiger/wiredtiger/blob/fd077b7479b8f04e62bb16e6d2aca9d7bbb1354a/test/format/ops.c#L1158) which sets up the truncate operation, or directly before it which is setting up operations in general. The one exception is in `replay_adjust_key` which is also only called before we setup the truncate ranges.

>Also I see that the default value is TV(RUNS_ROW), which is the number of rows defined in the table config. Are these different?

These should be the same, but we have some comments indicating it can change:
```
    /* The final number of rows in the table can change, get a local copy of the starting value. */
    rows_current = TV(RUNS_ROWS);
```
Since this was the existing value used to verify entire mirrors tables I'm happy to assume this covers at least to the end of the table.